# Analysis of cotton histone modification datasets
Datasets generated by Dr. Rick Masonbrink 2015-2016
---

## Project description
To investigate relationships between histone modifications and gene expression, this study used chromatin immunoprecipitation followed by high-throughput sequencing (ChIP-Seq) to profile the genome-wide distribution of histone acylation (H3K9ac) in cotton species. Young leaves from 9 *Gossypium* species (A1, A2, D1, D3, D5, AD1, AD2, AD3, AD4) and two outgroups (Kirkii, Kokia) were havested and used for cross-liking Chromatin Immunoprecipitation (ChIP) with anti-acetyl-histone H3 (Lys 9) (H3K9ac) antibodies. The ChIPed DNA were sequenced using IlluminaHiSeqâ„¢ 4000 for paired-end 150 cycles by BGI.

In combination with RNA-seq gene expression analysis, below question may be asked: Is H3K9ac assiciated with active gene transcription? Are histone marks enriched in genic regions? What about the disttibution profiles around transcription start sites (TSSs) of genes? Can these patterns assist us to understand gene expression evolution during cotton genus specification and polyploidization?

## Data preprocessing

### Raw fastq files
    cd ~/jfw-lab/RawData/Rick_Chip/clean_reads
    # ig - negative control; ac - H3K9ac; ch3 - cenH3
    ls *ac

### Previous analytic pipeline built by Rick
Bash pipeline (`~/jfw-lab/home/remkv6/bin/sickle2genomes1.2.sh`) processes raw fastq files:
* Quality filter and trim Fastq files - sickle
* Map reads to chloroplas, mitocondrial and ribosomal DNAs -  bwa
* Extract unmapped reads as new fastq files - samtools view, beadtools bam2fastq
* Map filtered reads to reference genome -bwa
* Call histone modification peaks - macs

---


